modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -95,6 +96,7 @@ def post_appointment(request_object_body)
           new_appointment = response.body
           convert_appointment_time(new_appointment)
           find_and_merge_provider_name(new_appointment) if new_appointment[:kind] == 'cc'
+          reason_code_service.extract_reason_code_fields(new_appointment)
```

Comment:
The scope of this ticket is to only extract the reason code field for VA appointment requests if appointment.kind == 'cc', we shouldn't extract reasonCode

Reply:
Updated the logic in `extract_reason_code_fields`